recipes/htpolynet/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "HTPolyNet" %}
+{% set version = "1.0.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/htpolynet-{{ version }}.tar.gz
+  sha256: fad2cd0699cf3a18537bf77962625ae1fbc669a266d698c4c51b49ce7d72bb59
+
+build:
+  entry_points:
+    - htpolynet = HTPolyNet.driver:cli
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [win or osx]
```

Comment:
I think this one can be `noarch`. What are the reasons to skip Windows and macOS?

Reply:
It will not run on Windows because it uses ambertools.  In theory it should work on OSX but I have not tried this myself.

Reply:
It is better to do noarch, not install on Windows due to a missing dependency, than making it "arch'ed" and build for several Python versions. Also, if ambertools is ever available on Windows, this package would be ready without the need for any change here.

Reply:
Here. on staged-recipes, you don't need to avoid it. Just let it error out. B/c it will be a noarch package only the Linux build will count and that is the only one that will appear in the feedstock when we merge this.

Reply:
The recipe I just pushed fails in a local linux build for a reason I can't figure out, which I quote in the commit message.